### ISSUE TITLE
YourMixer Hotkey invocation bug

### ISSUE THREAD

**Issue by Tethgar** [Jax-Core/YourMixer#14]
95% of the time when I try to invoke the YourMixer hotkey, the program will inevitably fail to open the mixer or will cause a massive lag spike that causes mouse movement to stutter for ~5 seconds. Also, switching outputs in the mixer doesn't work at all when switching outputs within the same audio device (ex. output 1-2 and output 3-4 on an Arturia audio interface).

Actually, the hotkeys pretty much don't work on anything, seems completely borked. IdleStyle has this issue plaguing hotkey invocation as well. Also Plainext causes micro stuttering when loaded. What's the point in running off of Rainmeter? It seems very poorly suited for the features in this app lol. Had all of the same issues 8 months ago and was pulling my hair out until I realized JaxCore was the root of all my problems, disappointing that these issues are still prevalent because I'd like to use it lol.

**Comment by LoveSempa** [comment]
I completely agree with you, there are a lot of joints and holes in this program, but as an idea it is not very bad, but it has not been brought to mind :/

**Comment by fallsdevil** [comment]
I'm also having problems with yourmixer, it keeps flashing on the screen and doesn't work using a hotkeys, just a widget...
